### DISCUSSION TITLE
Stereo mode for LAME MP3

### DISCUSSION THREAD

**Discussion by bozzis** [enzo1982/freac#329] [Help & Support] [answered]
![image](https://user-images.githubusercontent.com/24719518/174410837-2cc3d40f-90f4-4912-bc97-81bb263305d5.png)

Is it okay to keep it in "Auto" for quality purposes?
What does Auto do exactly? Does it analyze if it's gonna use Joint Stereo vs Stereo, then picks it, for each song?

**Comment by enzo1982** [comment, marked as answer]
_Auto_ means to use the LAME encoder's default stereo mode. This is currently equivalent to _Joint Stereo_ and is unlikely to ever change again in the future.

In _Joint Stereo_ mode, the LAME encoder decides for each frame (1152 samples) whether to use Stereo or Joint Stereo. I.e. for audio sampled at 44.1 kHz, the encoder makes that decision almost 40 times per second.

_Auto_ or _Joint Stereo_ is the recommended setting for best encoding quality. It is not recommended to enable the _Force Joint Stereo_ option. See [here](https://www.esm.rochester.edu/ears/docs/lame/switchs.html#m) for more details on the different stereo modes.
